Week3/homework/js-exercises/5_TotalCost.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+function calculateTotalPrice(number) {
+    return Object.values(cartForParty).reduce((drinks, chips) => drinks + chips);
```

Comment:
👏 for using `Object.values` and `reduce` but the parameter names used inside `reduce` aren't correct because the items are not restricted to just `drinks` and `chips`. The first parameter is an accumulator i.e `total` which you need to calculate while the second parameter is the item from the array itself. 🙂